test/graaflib/algorithm/clique/bron_kerbosch_test.cpp

```diff
@@ -0,0 +1,299 @@
+#include <fmt/core.h>
+#include <graaflib/algorithm/clique/bron_kerbosch.h>
+#include <graaflib/graph.h>
+#include <graaflib/types.h>
+#include <gtest/gtest.h>
+
+#include <random>
+
+namespace graaf::algorithm {
+namespace {
+template <typename T>
+struct BronKerbosch : public testing::Test {
+  using graph_t = T;
+};
+
+using graph_types = testing::Types<undirected_graph<int, int>>;
+TYPED_TEST_SUITE(BronKerbosch, graph_types);
```

Comment:
[minor]: If we only have one graph type we are testing, there is no real need for a `TYPED_TEST_SUITE`, instead we could just write the type directly in the tests. However, if you do want to test this for different graph types, we now have predefined type collections you can use here in `<utils/fixtures/fixtures.h>`.